src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const populationTotal = Array.from(population).reduce((acc, country) => {
+  const number = +country.textContent.split(',').join('');
+
+  return acc + number;
+}, 0);
+
+totalPopulation.textContent = populationTotal.toLocaleString();
+
+averagePopulation.textContent = (
+  populationTotal / population.length
```

Comment:
There is a potential division by zero error if there are no elements with the class 'population'. Consider adding a check to ensure that `population.length` is not zero before performing the division.